src/components/calcite-button/calcite-button.tsx

```diff
@@ -103,13 +106,19 @@ export class CalciteButton {
     const iconStartEl = (
       <calcite-icon
         class="calcite-button--icon icon-start"
+        flip-rtl={this.iconFlipRtl === "start" || this.iconFlipRtl === "both"}
```

Comment:
This should use the prop name: `flipRtl`. This applies to other similar lines in this PR.

Reply:
Hm, but since its being set on `<calcite-button`>, doesn't the `icon-` prefix help infer that the icon is what is getting flipped in rtl? If it were just `flip-rtl`, it makes it seem like the button will get flipped in rtl.

Reply:
No, I mean this, and similar lines, should be `flipRtl={this.iconFlipRtl === "start" || this.iconFlipRtl === "both"}`.

Reply:
Oh, sorry misunderstood. Will update.